proto/rig/operator/api/v1/cluster/service.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package api.v1.operator.cluster;
+
+import "model/metrics.proto";
+
+service Service {
+  rpc GetNodes(GetNodesRequest) returns (GetNodesResponse) {}
+  rpc GetPodsOfNode(GetPodsOfNodeRequest) returns (GetPodsOfNodeResponse) {}
+  rpc GetNodeUsages(GetNodeUsagesRequest) returns (GetNodeUsagesResponse) {}
+}
+
+message GetNodesRequest {}
+
+message GetNodesResponse {
+  repeated Node nodes = 1;
+}
+
+message Node {
+  string node_name = 1;
+  model.Resources allocateable = 2;
+  uint64 max_pods = 3;
+}
+
+message GetPodsOfNodeRequest {
+  string node_name = 1;
+}
+
+message GetPodsOfNodeResponse {
+  repeated Pod pods = 1;
+}
+
+message Pod {
+  string pod_name = 1;
+  string namespace = 2;
+  model.Resources requested = 3;
+  string capsule_name = 4;
+}
+
+message GetNodeUsagesRequest {}
+
+message GetNodeUsagesResponse {
+  repeated NodeUsage nodes = 1;
+}
+
+message NodeUsage {
+  string node_name = 1;
+  model.Resources usage = 2;
```

Comment:
would it simplify or make it more complex, if this was moved to the Node message instead?

Reply:
It would simplify stuff on the caller side, but then we would always be calling the metric server when getting nodes. But come to think of it, when we list nodes in the platform, we always immediately follow up with getting node usages.